Formula/rust.rb

```diff
@@ -87,19 +76,6 @@ def install
       system "cargo", "install", "--root", prefix, "--path", ".", "--features", "curl-sys/force-system-lib-on-osx"
     end
 
-    resource("racer").stage do
-      ENV.prepend_path "PATH", bin
-      cargo_home = buildpath/"cargo_home"
-      cargo_home.mkpath
-      ENV["CARGO_HOME"] = cargo_home
-      system "cargo", "install", "--root", libexec, "--path", "."
-      (bin/"racer").write_env_script(libexec/"bin/racer", :RUST_SRC_PATH => pkgshare/"rust_src")
-    end
-
-    # Remove any binary files; as Homebrew will run ranlib on them and barf.
-    rm_rf Dir["src/{llvm-project,llvm-emscripten,test,librustdoc,etc/snapshot.pyc}"]
-    (pkgshare/"rust_src").install Dir["src/*"]
```

Comment:
I removed this, since the source was seemingly retained only for Racer. If this was included for other reasons, I can add it back. If anyone creates a Racer formula in the future, they would need to restore this part but it would be better if it could be made opt-in via a flag, rather than making the installation larger by default for all users.

Reply:
I think it is fine!

Reply:
We can revisit this as we go.